pom.xml

```diff
@@ -271,6 +271,8 @@
     </dependencyManagement>
 
     <modules>
+        <module>kms</module>
```

Comment:
In  #710 we'd talked about (and agreed, I thought) prefixing module names consistent with `kroxylicious-`.     We should adopt this convention in this new work.

Reply:
That would be more consistent, yes. And it's what I've done in #710. But you don't half get fed up with writing kroxylicious everywhere, and it feels redundant when kroxylicious is in the `groupId`.

Reply:
Yeah, I flip flop on it being helpful in the artefact name. 

I think I come down on the `kroxylicious-` prefix being more helpful than not.

Reply:
OK, I can change it. Note that means we'll have `kroxylicious-kms-provider-kroxylicious-inmemory` as an artifact name.

Reply:
yep I imagine it being helpful later on, being able to grep out the versions of all 'kroxylicious-*' dependencies on disk or in your image.

Reply:
>OK, I can change it. Note that means we'll have kroxylicious-kms-provider-kroxylicious-inmemory as an artifact name.

Why does it have the 2nd kroxylicious? `kroxylicious-kms-provider-inmemory`  seems ok, `kroxylicious-inmemory-kms-provider` sounds bit more readable.